examples/preprocessing/plot_muscle_detection.py

```diff
@@ -0,0 +1,85 @@
+"""
+===========================
+Annotate muscle artifacts
+===========================
+
+Muscle contractions produce high frequency activity that can contaminate the
+brain signal of interest. Muscle artifacts can be produced when clinching the
+jaw, swalloing, or twitching a head muscle. Muscle artifacts are most notable
+in the range of 110-140Hz.
+
+This example uses `annotate_muscle` to annotate segments where muscle activity
+likely occurred. This is done by band-pass filtering the data in the 110-140 Hz
+range. Then, the envelope is taken to account for peaks and troughs. The
+envelope is z-scored and averaged across channels. To remove noisy transient
+peaks, the z-scored average is low-pass filtered to 4 Hz. Segments above a set
+threshold are annotated as BAD_motion. In addition, `min_length_good` allows to
+discard god segments of data between bad segments that are to transient.
+
+Note:
+The raw data needs to be notched to remove line activity, otherwise, changes of
+AC power could bias the estimation.
```

Comment:
```suggestion
.. note::
    If line noise is present, you should perform notch-filtering *before*
    detecting muscle artifacts. See :ref:`tut-section-line-noise` for an
    example.
```

You will need to add a cross-reference target `.. _tut-section-line-noise` just above line 104 of `tutorials/preprocessing/plot_30_filtering_resampling.py` (just before the title of the "power line noise" section).